sdc/rabbit/async_consumer.py

```diff
@@ -0,0 +1,387 @@
+import logging
```

Comment:
Is there a way to pass in logging formatting et al to the library (without relying on `sdx-common` which a consumer may not want to also pull in just for logging format and they may have a differing requirement)?